neurots/schemas/distributions.json

```diff
@@ -74,7 +117,7 @@
                             "type": "null"
                         },
                         {
-                            "additionalProperties": false,
+                            "additionalProperties": true,
```

Comment:
why this changed?

Reply:
I will clean these last, I was trying to quickly fix on the fly, but failed so far

Reply:
Let's keep this unresolved for now to not forget.

Reply:
ah yes, I added the missing entries in the schema